lib/src/__tests__/e2e/DatePickerProps.test.tsx

```diff
@@ -77,4 +78,29 @@ describe('DatePicker - different props', () => {
     expect(onCloseMock).toHaveBeenCalled();
     expect(onChangeMock).toHaveBeenCalled();
   });
+
+  it('ref - should forwardRef to text field', () => {
```

Comment:
We have started to normalize the common expectations from the components in https://github.com/mui-org/material-ui/blob/master/packages/material-ui/src/test-utils/describeConformance.js. It might help in the future.